src/main/java/httpserver/HttpResponse.java

```diff
@@ -0,0 +1,5 @@
+package httpserver;
+
+public class HttpResponse{
+    public Integer status = 0;
```

Comment:
Just curious, does this need to be an integer?

Reply:
No. I thought about making it a `String`, but since all the statuses are numbers and the `Integer` class is automatically cast to a string when appropriate; so I thought why not.